teuthology/__init__.py

```diff
@@ -14,10 +14,22 @@
 
 import logging
 import os
+import subprocess
 
+mydir = os.path.dirname(os.path.realpath(__file__))
```

Comment:
This is pedantic, but why store this in a variable? Could just put it down on line 28